src/Application/Migrations/15_function_ticket_state.sql

```diff
@@ -96,22 +114,31 @@ END
 $$
 LANGUAGE plpgsql;
 
-CREATE OR REPLACE FUNCTION ticket_state_commence(param_project_id bigint, param_ticket_type enum_ticket_type)
+CREATE OR REPLACE FUNCTION ticket_state_commence(param_ticket_id bigint)
   RETURNS enum_ticket_state AS
 $$
 DECLARE
+  var_project_id bigint;
+  var_ticket_type enum_ticket_type;
   ret enum_ticket_state;
   next_state record;
 BEGIN
+  SELECT
+    project_id, ticket_type INTO var_project_id, var_ticket_type
+  FROM
+    tbl_ticket t
+  WHERE
+    t.id = param_ticket_id;
```

Comment:
What happens if that ticket_id is not found?

Reply:
Then ticket_state_initial(NULL, NULL) is called which results in

https://github.com/crs-tools/tracker/blob/84016338c8636a97e9adec3c6cf37075e1a087dd/src/Application/Migrations/15_function_ticket_state.sql#L108

which returns NULL, so the while loop is not entered and function returns NULL.

Reply:
So, nothing breaks but the warning message is misleading. Mh.
Maybe adding a `IF NOT FOUND THEN` clause might be worth it.